p2p/conn/connection.go

```diff
@@ -452,8 +480,10 @@ FOR_LOOP:
 			c.Logger.Debug("Receive Ping")
 			c.pong <- struct{}{}
 		case packetTypePong:
-			// do nothing
 			c.Logger.Debug("Receive Pong")
+			if !c.pongTimer.Stop() {
```

Comment:
what if we receive multiple pongs in a row ? wont we stop on the first one, and then this will return false and we'll block forever ?